joblib/pool.py

```diff
@@ -363,14 +351,12 @@ def get():
 
         self.get = get
 
-        if self._reducers:
```

Comment:
It seems like this just adds a method to the `self` instance. Still I would avoid changes like this unless there is a good reason for it.